.gitignore

```diff
@@ -86,3 +86,4 @@ prisma
 # next.js
 .next
 /public/sw-env.js
+.env.local
```

Comment:
we don't expect this, `.env` is a context

Reply:
removed, done